src/library/export/coverartcopyworker.cpp

```diff
@@ -9,7 +9,16 @@
 #include "util/safelywritablefile.h"
 
 void CoverArtCopyWorker::run() {
-    auto selectedCover = mixxx::FileAccess(mixxx::FileInfo(m_selectedCoverArtFilePath));
+    auto coverArtFileInfo = mixxx::FileInfo(m_selectedCoverArtFilePath);
+    auto selectedCoverFileAccess = mixxx::FileAccess(coverArtFileInfo);
+    if (!selectedCoverFileAccess.isReadable()) {
+        if (Sandbox::askForAccess(&coverArtFileInfo)) {
```

Comment:
This will issue a message box from this thread. I am not an apply guy. Is this necessary at all?

Reply:
Oh, I didn't notice that the askForAccess populates message box. That will cause a SEG fault according to my experience with this worker. So I just check if it is readable, if it is not readable emitting fail signal would work IMHO. What do you think?